web/src/components/ZWebsiteNav/ZWebsiteNav.vue

```diff
@@ -26,6 +26,21 @@
           width='56px',
           :color='logoColor'
         )
+      .z-website-nav__logo__content(v-else)
+        z-link.z-website-nav__logo__wrapper(:href='logoHref')
+          z-logo.z-website-nav__logo.z-website-nav__logo--desktop(
+            outline,
+            width='84px',
+            :color='logoColor'
+          )
+          z-logo.z-website-nav__logo.z-website-nav__logo--mobile(
+            outline,
+            width='56px',
+            :color='logoColor'
+          )
+        .z-website-nav__separator(v-if="breadcrumb")
+        z-link.z-website-nav__logo.z-website-nav__logo--blog(:href='breadcrumbLink')
```

Comment:
Wrong semantics of class name : this must not be explicitely related to `blog` (the breadcrumb can be anything else). This also must not be considered as a "logo". Create a new class `.z-website-nav__breadcrumb`:
 ```suggestion
         z-link.z-website-nav__breadcrumb(:href='breadcrumbLink')
```